File: Find the Town Judge.java
class Solution {
    public int findJudge(int n, int[][] trust) {
        int[] trusted = new int[n + 1];

        for (int[] pair : trust) {
            trusted[pair[0]] -= 1;
            trusted[pair[1]] += 1;
        }

        for (int i = 1; i <= n; i++) {
            if (trusted[i] == n - 1) {
                return i;
            }
        }

        return -1;        
    }
}

File: Find the Town Judge.py
class Solution:
    def findJudge(self, n: int, trust: List[List[int]]) -> int:
        trusted = [0] * (n + 1)

        for a, b in trust:
            trusted[a] -= 1
            trusted[b] += 1
        
        for i in range(1, n + 1):
            if trusted[i] == n - 1:
                return i
        
        return -1
